packages/legend-query-builder/src/components/QueryBuilderConstantExpressionPanel.tsx

```diff
@@ -121,28 +131,33 @@ const QueryBuilderSimpleConstantExpressionEditor = observer(
           queryBuilderState.graphManagerState.graph,
           val.value,
           queryBuilderState.observerContext,
+          queryBuilderState.INTERNAL__enableInitializingDefaultSimpleExpressionValue,
         );
         setSelectedValue(newValSpec);
       }
+      setShouldFocusOnValue(true);
     };
+    const supportedPrimitiveTypes: PrimitiveType[] = [
```

Comment:
lets add a comment here on the ones we are disabling

Reply:
Done